spec/classes/init_spec.rb

```diff
@@ -268,6 +269,27 @@
     end
   end
 
+  describe 'with \'fqdn_ip_address\' parameter set to' do
+    context 'an invalid type (not ip address)' do
```

Comment:
There should also be a positive test, so when fqdn_ip_address is set to a different ip that `$::ipaddress` and a test for when fqdn_ip_address is set to the same ip as `$::ipaddress`